src/main/java/com/thealgorithms/bitmanipulation/SingleBitOperators.java

```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.bitmanipulation;
+
+/*
+ * Author: lukasb1b (https://github.com/lukasb1b)
+ */
+
+public class SingleBitOperators {

```

Comment:
Let's make this class a _proper utility class_.

```suggestion
public final class SingleBitOperators {
    private SingleBitOperators() {
    }
```